SETUP/upgrade/14/20200404_normalize_project_details.php

```diff
@@ -0,0 +1,69 @@
+<?php
+$relPath='../../../pinc/';
+include_once($relPath.'base.inc');
+include_once($relPath.'misc.inc');
+include_once($relPath.'unicode.inc');
+
+header('Content-type: text/plain');
+
+// ------------------------------------------------------------
+
+echo "Normalizing project details...";
+
+$sql = "
+    SELECT projectid, nameofwork, authorsname, comments, extra_credits
+    FROM projects
+    ORDER BY projectid;
+";
+
+$result = mysqli_query(DPDatabase::get_connection(), $sql) or die( mysqli_error(DPDatabase::get_connection()) );
+
+$projects = array();
+while($row = mysqli_fetch_assoc($result))
+{
+    $projects[] = $row;
+}
+
+$stop_file = '/tmp/utf8_normalization';
+
+echo <<<EOF
+This will now start normalizing all of the project fields.
+This will likely take a while.
+To stop the process gracefully create $stop_file
+(eg: touch $stop_file) and it will stop after it
+finishes the table it is working on.
+
+EOF;
+
+echo count($projects) . " possible projects to normalize.\n";
+
+foreach($projects as $project)
+{
+    if(file_exists($stop_file))
```

Comment:
Would be nice to output the projectid being worked on in the loop so we can see progress as it goes.